### ISSUE TITLE
Capture total GC pause from sync gateway expvars

### ISSUE THREAD

**Issue by tleyden** [couchbase/sync_gateway#633]
As described in this issue: 

https://github.com/couchbase/sync_gateway/issues/617 

there are problems with long GC pauses. 

After the sync gateway expvar is collected, we should capture the totalPauseNS field: 

https://gist.github.com/tleyden/a6c4935b3405f3e37595#file-gistfile1-txt-L201 

for now, just output to sgw_test_info_0.txt and the console (the value should be converted to seconds before being output, so it's human readable). Later, we will probably need to add some threshold values that will signal a performance failure if this value is too high.

**Comment by tleyden** [comment]
No longer needed after #688
